src/views/election_info.hbs

```diff
@@ -187,7 +187,7 @@
 
     <div class="col">
       <p>Die ausführlichen Antworten zu Frage 6 können hier heruntergeladen werden.</p>
-      <a href="/download/SelbstbestimmteBildung_Wahlprüfsteine_2021.pdf" download><button class="btn btn-secondary mt-2 mb-3">Antworten als pdf</button></a>
+      <a href="/download/SelbstbestimmteBildung-Wahl-2021.pdf" download><button class="btn btn-secondary mt-2 mb-3">Antworten als pdf</button></a>
```

Comment:
I'd recommend to prefer all lowercase filenames, because macOS and Windows are case-insensitive, but linux (and we host on linux) isn't.